Quantum-Control-Applications/Superconducting/Kerr-Cat SNAIL-mon/Use Case 2 - QuLAB - Kerr-Cat/cqr_blobs_yale.py

```diff
@@ -0,0 +1,110 @@
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from scipy.optimize import minimize
+
+from qm import SimulationConfig
+from configuration import *
+import matplotlib.pyplot as plt
+import numpy as np
+import matplotlib as mpl
+
+###################
+# The QUA program #
+###################
+
+# %matplotlib qt
+
+n_runs = 10000
+
+cooldown_time = 5 * qubit_T1 // 4
+
+with program() as IQ_blobs:
+    n = declare(int)
+    I_g = declare(fixed)
+    Q_g = declare(fixed)
+    I_g_st = declare_stream()
+    Q_g_st = declare_stream()
+    I_e = declare(fixed)
+    Q_e = declare(fixed)
+    I_e_st = declare_stream()
+    Q_e_st = declare_stream()
+    i = declare(int)
+    
+    I = declare(fixed)
+    Q = declare(fixed)
+    I_st = declare_stream()
+    Q_st = declare_stream()
+    
+    I_th = declare(fixed, value=0.0)
+    
+    adc_st = declare_stream(adc_trace=True)
+
+    # frame_rotation_2pi(0.0, 'squeeze_rise','squeeze_drive','squeeze_fall')
+    # frame_rotation_2pi(0.25, 'cqr_drive')
+
+    with for_(n, 0, n < n_runs, n + 1):
+        
+        # update_frequency('squeeze_drive', int(0))
+        # update_frequency('squeeze_rise', int(0))
+        # update_frequency('cqr_drive', int(0))
+        # update_frequency('resonator', int(0))
+        
+        
+        play('on', 'squeeze_switch', duration=int((4e6+1e3)//4))
+        play('ftc_rise', 'squeeze_rise')
+        align('squeeze_rise', 'squeeze_drive')
+        play('cw', 'squeeze_drive', duration=int(4e6//4))
+        align('squeeze_drive', 'squeeze_fall')
+        play('ftc_fall', 'squeeze_fall')
```

Comment:
Here you split the pulse into three parts because you want to control the flat length without stretching the rise and fall parts, right?
If you do it with a single element you end up having gaps?